src/vs/platform/quickinput/browser/quickInputList.ts

```diff
@@ -539,7 +539,7 @@ export class QuickInputList {
 			}
 		}));
 
-		const delayer = new ThrottledDelayer(options.hoverDelegate.delay);
+		const delayer = new ThrottledDelayer(options.hoverDelegate ? options.hoverDelegate.delay : 200);
```

Comment:
All this stuff with the `delayer` and `this.showHover` is only going to happen if there is a `hoverDelegate`. Instead of doing the check for `this.options.hoverDelegate` in all these places, can you just not do any of this stuff if `this.options.hoverDelegate` isn't defined?